gameplay/src/Quaternion.h

```diff
@@ -231,6 +252,18 @@ class Quaternion
      */
     void normalize(Quaternion* dst) const;
 
+	/**
+	* Rotate the specified point by this quaternion
+	* and stores the result in dst
+	*
+	* Note: The point must normalized.
```

Comment:
Here I think you mean that the quaternion must be normalized(?) I think the vector that represents the point can be any length.